perllib/FixMyStreet/Roles/ParallelAPI.pm

```diff
@@ -35,7 +42,10 @@ sub call_api {
     } else {
```

Comment:
I think we can check if $tmp exists here and get it if it does, bypassing calling the external script at all.